src/sql/platform/connection/common/connectionConfig.ts

```diff
@@ -375,6 +375,15 @@ export class ConnectionConfig {
 		return result;
 	}
 
+	private checkIfAuthenticationOptionsMatch(profileStore: IConnectionProfileStore, profile: ConnectionProfile): boolean {
+		if ((profileStore.options.authenticationType === undefined || profileStore.options.authenticationType === '') && (profile.options.authenticationType === undefined || profile.options.authenticationType === '')) {
+			return true;
+		}
+		else {
+			return profileStore.options.authenticationType === profile.options.authenticationType;
+		}
```

Comment:
I don't think authentication type should be undefined/empty - it should ideally be a value from 'Enum'..

Reply:
Yeah you're right about that, the ones in the test had them as empty.